.github/ISSUE_TEMPLATE/new_vm.md

```diff
@@ -0,0 +1,38 @@
+---
+name: Request a new VM
+about: Assemble the necessary information to request a new VM
+title: ''
+labels: ''
+assignees: ''
+
+---
+### Describe the VM(s) you need
+
+If you do not know the answers to some of these questions, that's okay. Please add the information you do have, and we will work together to gather any other information we need.
```

Comment:
you could also add a link to the new vm documentation for more details (unless that is going away)

Reply:
good idea, added in 36f24aaeb1cdcd84a553c340a0393e6409c17de1